estimagic/visualization/comparison_plot.py

```diff
@@ -0,0 +1,440 @@
+"""
+Draw an interactive comparison plot of named result dictionaries.
+
+The plot can plot many results for large numbers of parameters
+against each other.
+
+The plot can answer the following questions:
+
+1. How are the parameters distributed?
+
+2. How large are the differences in parameter estimates between results
+    compared to the uncertainty around the parameter estimates?
+
+3. Are parameters of groups of results clustered?
+
+"""
+import numpy as np
+import pandas as pd
+from bokeh.layouts import gridplot
+from bokeh.models import BoxSelectTool
+from bokeh.models import ColumnDataSource
+from bokeh.models import HoverTool
+from bokeh.models import TapTool
+from bokeh.models import Title
+from bokeh.models.callbacks import CustomJS
+from bokeh.models.widgets import CheckboxGroup
+from bokeh.plotting import figure
+from bokeh.plotting import show
+
+from estimagic.optimization.utilities import index_element_to_string
+
+
+def comparison_plot(
+    res_dict, color_dict=None, height=None, width=600, point_estimate_plot_kwargs=None
```

Comment:
Why is width=600? Are these the normal matplotlib sizes in inches or works bokeh differently?

Reply:
In bokeh plot sizes are given in points, not inches. I improved on the default width and height in 499d9ad7f8ee2c909fffeaa16095c6265f9ca048.